addons/website_slides/tests/test_attendee.py

```diff
@@ -94,6 +94,53 @@ def test_enroll_to_course(self):
         # Subscribe enrolled attendees to the chatter
         self.assertTrue(user_portal_partner.id in self.channel.message_partner_ids.ids)
 
+    @users('user_officer')
+    def test_is_member_is_member_invited_search_on_channel(self):
```

Comment:
Isn't it already covered by use cases in "test_security.py" ? Just to be sure we don't inflate tests too much (how dare you say that TDE ?) ?

Reply:
Indirectly, but you are right I think, I will remove this innocent test then!